### ISSUE TITLE
Verify loop mass flow rates are handled properly once the condenser code is merged with the plant simulation code. (CR #7655)

### ISSUE THREAD

**Issue by axelstudios** [NREL/EnergyPlus#2436]
## 
#### Description

2008-12-22 ESL:

Uses file 7578-\* (Constant speed water to air heat pumps attached directly to condenser loop) 

To reproduce: Set loop flow rate max flow rate < constant speed pump flow rate

Error is not signaled when this is encountered, and bizarre behavior is encountered (CR 7558, 7578).  It is expected that the plant flow resolver would recognize this, so once code is merged, verify this.  If not, find a fix for it.  The actual problem steps are laid out here as a story in my notes:

```
Input file has condenser loop: 
    0.0033,              !- Maximum Loop Flow Rate {m3/s}
And pump:
    0.004,                    !- Rated Flow Rate {m3/s}
So the pump was overpowering the loop and not caring at all about a max flow rate.
The max, however, was still passed around
And down in UpdateMixer, the mixer outlet maxavail was constrained down to 3.3
This was passed all around the demand side of the loop then
So 4kg/s was trying to be passed around a loop with 3.3 maxavail
Then the demand side splitter would give the maxavail to the bypass (3.3)
And have 0.7 to deliver and so it forced it on the last HP
```

[BG 12-23-2009]  transitioned up test file from 7578 and it produced a hard crash at line 1228 in HVACWaterToAir.f90.  Flipping this CR back to open to deal with this, although the current hard crash may be a separate issue.  added defect file that has been transitioned to version 5. 

7655_in.idf 

2010-01-08 ESL: Some of this is very close to some of the issues encountered in CR 7745, in which branch pumps were causing a flow > max flow to be passed around the loop.  Will start looking into the hard crash now.

2010-01-08 ESL: The hard crash was an input problem.  Two of the coils (system 2 heating and cooling) had not been transitioned up to the new parameter estimation wahp coil syntax.  Thus the UA value was being read in as zero, and causing a hard crash.

2010-0109 LKL: as checked in the code fataled out in two of the test suite files.  V4 - V5 transition has not been done (written) but there are changes due to breaking out of some of the parameter fields (I think) for ease of IP units in the IDF Editor.  Reopening and will reset code back to before checkin.  Hand transition to V5 must be accomplished?

2010-01-13 ESL: The error handling required a touch more detail to ensure it only fatals out when necessary.  New HVACWaterToAir.f90 was checked in that DOES NOT fatal the test suite files, but properly DOES fatal the defect file.
## 
#### Fix

2009-08-31 ESL: The fix was accounted for in the condenser loop merge, but a warning message was still added to assist users in fixing the problems when the branch flow rate turns out to be greater than the loop maximum.  Obviously something is driving the flow too high, and most likely it is the pump.  Either they need to increase the loop max flow or turn down their pump.

2010-01-08 ESL: The crash in the wahp was an input issue.  By copying the parameters from system 1 heating and cooling coils into the system 2 coils, the simulation runs perfect.
## 

Addressed in `10.01.17 V5.0.0.013` by @Myoldmopar
Last build tested: `08.12.19 V3.1.0.004`
